web/app/themes/ppj/src/sass/components/video-player.sass

```diff
@@ -40,6 +40,7 @@
 
   &__cover-image
     height: 100%
+    object-fit: cover
```

Comment:
I don't think `object-fit: cover` is required here. (Ditto for the `object-fit` polyfill)

Reply:
We need object fit until we are using the correct image sizes for the cover images, the current images aren't strictly 16:9 ratio (but they are quite close). I can update the video player image sizes in whatever branch I'm making the other object-fit changes (ie11 branch now presumably), and then I can remove this css rule at the same time.

Reply:
Done in this branch in the end.